DigitalLearningSolutions.Web/Controllers/Signposting/ViewResourceController.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.Signposting

```

Comment:
There was a controller added under HEEDLS-681. This here could be combined into that controller: `SignpostingSsoController`